### PR TITLE
Build and run in Github Actions

### PR DESCRIPTION
**Pull request by harlantwood**

### CODE REVIEW

File: tests/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "TRYORAMA_LOG_LEVEL=info RUST_BACKTRACE=1 RUST_LOG=holochain::core::ribosome::host_fn::debug=debug TRYORAMA_HOLOCHAIN_PATH=\"holochain\" node --loader ts-node/esm --experimental-specifier-resolution=node src/index.ts | tap-diff"
+    "test": "set -o pipefail && TRYORAMA_LOG_LEVEL=info RUST_BACKTRACE=1 RUST_LOG=holochain::core::ribosome::host_fn::debug=debug TRYORAMA_HOLOCHAIN_PATH=\"holochain\" node --loader ts-node/esm --experimental-specifier-resolution=node src/index.ts | tap-diff"
```
Comment (harlantwood) [inline tests/package.json:7]: Fail if any part of test command fails

Comment (harlantwood) [reply tests/package.json:7]: Without this, if the test command fails, `tap-diff` still returns zero error code, so CI is "green".

Comment (jost-s) [reply tests/package.json:7]: We could remove tap-diff from the scaffolding or add this setting. @guillemcordoba

Comment (harlantwood) [reply tests/package.json:7]: Good call, opened PR there: https://github.com/holochain/scaffolding/pull/6

**Pull request merged**